### ISSUE TITLE
Variable viewer: Support keyboard accessibility

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-jupyter#10233]
Testing #10170

The column headers should be focusable by tab and toggle via enter

**Comment by roblourens** [comment]
https://github.com/microsoft/vscode/issues/165445
